src/client/App.js

```diff
@@ -31,4 +32,8 @@ const App = () => {
 	);
 };
 
-export default App;
+const mapStateToProps = (state) => {
+  return {state};
```

Comment:
(feed: state.feed) so you don't return the whole state tree and just the feed section which you need for the results components